cognite/neat/rules/models/asset/_validation.py

```diff
@@ -1,4 +1,41 @@
+from graphlib import CycleError
+from typing import cast
+
+from cognite.neat.rules import issues
+from cognite.neat.rules.issues.base import IssueList
+from cognite.neat.rules.models._base import SheetList
+from cognite.neat.rules.models.asset._rules import AssetProperty, AssetRules
+from cognite.neat.rules.models.entities import AssetEntity, AssetFields, ClassEntity
 from cognite.neat.rules.models.information._validation import InformationPostValidation
 
 
-class AssetPostValidation(InformationPostValidation): ...
+class AssetPostValidation(InformationPostValidation):
+    def validate(self) -> IssueList:
+        self.issue_list = super().validate()
+        self._parent_property_point_to_class()
+        self._circular_dependency()
+        return self.issue_list
+
+    def _parent_property_point_to_class(self) -> None:
+        compromised_class_property = []
```

Comment:
Is there a less generic more descriptive name here? What is compromised in this context?